lib/screens/loginPage.dart

```diff
@@ -1,5 +1,7 @@
 import 'package:flutter/material.dart';
```

Comment:
we have a file called "screens/loginPart.dart" and we have a file called "LoginPage.dart"

this is not correct at all.

can we follow this https://medium.com/flutter-community/flutter-code-organization-de3a4c219149

Reply:
also all filename should be lower case only with underscore if needed.

Reply:
this is resolved as well